pkg/network/namescheme/networknamescheme.go

```diff
@@ -134,6 +134,18 @@ func CreateNetworkNameSchemeByPodNetworkStatus(networks []v1.Network, networkSta
 	return CreateHashedNetworkNameScheme(networks)
 }
 
+// CreateFromNetworkStatuses creates a mapping of network name to pod interface name
+// based on the given VMI spec networks and pod network statuses.
+// In case the pod network status list has at least one interface with ordinal interface name -
+// the function will return an ordinal network name-scheme.
+func CreateFromNetworkStatuses(networks []v1.Network, networkStatuses []networkv1.NetworkStatus) map[string]string {
```

Comment:
The function name is hiding what it creates. Please rename it and not to `CreateNetworkNameSchemeByPodNetworkStatus2` :)

Reply:
When looking at an exported Go identifier, we need to remember that it is prefixed by its package name, so this specific function is actually called `namescheme.CreateFromNetworkStatuses` when called from other packages.
This is a recurring idiom in Go.

In case the name is still off, feel free to suggest another name.

You will be happy to see that one of the commits in this PR removed the `2` suffix from the function you were referring to.